### DISCUSSION TITLE
Just how do I get mermaid to show a graph?

### DISCUSSION THREAD

**Discussion by wf5w** [gollum/gollum#1865] [Q&A] [answered]
I just cloned gollum, and followed the instructions for installing and running.
I saw that mermaid was included and was ecstatic!

However, I cannot seem to get anything to work, I tried this:

\```mermaid
graph LR
    A[find] --> |stdout| B[grep for doc]
    B --> |stdout| C[grep for linux]
    C --> E(stdout)
\```

and noting but an error.

So, is there some other way to trigger this to happen? I don't see anything much in the wiki about it.

Thanks, in advance: great product, I will put this on my raspberry pi soon!!!!

**Comment by dometto** [comment]
Are you running gollum from `master` (e.g. via [Docker](https://github.com/gollum/gollum/wiki/Gollum-via-Docker))? Mermaid support hasn't been added to the latest release of the gem yet. Also be sure to run gollum with with `--mermaid` option! Otherwise what you're doing seems exactly right :)

**Comment by wf5w** [comment]
I did a git clone of gollum, but I did a gem install of gollum.

So I guess I should wait.

**Comment by dometto** [comment, marked as answer]
Or alternatively, run Gollum from Docker (using the `master` image): https://github.com/gollum/gollum/wiki/Gollum-via-Docker

Again, don't forget to use the `--mermaid` option.
